libs/nspr/Makefile

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (C) 2019 Lucian Cristian
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nspr
+PKG_VERSION:=4.21
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Lucian Cristian <lucian.cristian@gmail.com>
+PKG_LICENCE:=GPL-2.0+
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://archive.mozilla.org/pub/$(PKG_NAME)/releases/v$(PKG_VERSION)/src/
+PKG_HASH:=15ea32c7b100217b6e3193bc03e77f485d9bf7504051443ba9ce86d1c17c6b5a
+
+PKG_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+CONFIGURE_PATH = ./nspr
+MAKE_PATH = ./nspr
+PKG_AUTOMAKE_PATHS = $(PKG_BUILD_DIR)/nspr
+
+
+x64=--enable-64bit
+
+ifeq ($(ARCH),aarch64)
+        conf=$(x64)
+else ifeq ($(ARCH),mips64)
+        conf=$(x64)
+else ifeq ($(ARCH),ppc64)
+        conf=$(x64)
+else ifeq ($(ARCH),x86_64)
+        conf=$(x64)
+endif
+
+export MUSL=$(if $(CONFIG_LIBC_USE_GLIBC),0,1)
```

Comment:
this looks dodgy

Reply:
I need a way to pass some values if ipv6 and musl, otherwise it will fail to compile, this one I found it best